### ISSUE TITLE
CloneCleaner is reproducing the same prompt repeatedly

### ISSUE THREAD

**Issue by jackel321619** [artyfacialintelagent/CloneCleaner#19]
I have used this tool for many months, and in most cases it is flawless, however, when making an even minor change (adding text to neg prompts, changing CFG scale, etc,) the results of the next 5 images are as follows:
(Sadie from Australia, long [blonde:rose-gold:.2] hair:2), An unparalleled 8K UHD professional photograph...
(Sadie from Australia, long [blonde:rose-gold:.2] hair:2), An unparalleled 8K UHD professional photograph...
(Sadie from Australia, long [blonde:rose-gold:.2] hair:2), An unparalleled 8K UHD professional photograph....
(Sadie from Australia, long [blonde:rose-gold:.2] hair:2), An unparalleled 8K UHD professional photograph...
(Sadie from Australia, long [blonde:rose-gold:.2] hair:2), An unparalleled 8K UHD professional photograph...

All 5 photos have slight differences, but it is obviously the same individual with changes to hair, and that's about it.. 
I have tried multiple checkpoints, multiple samples, all with the same result. It seems CloneCleaner is getting hung up, and i don't know how to clean out the cache to force it back on track while to a batch of 10..

Before this glitch, i would get the following as a batch (as I understand this is how CloneCleaner is programed, and this is what i want back): 
(Evelyn from USA, long (waist-length:1.2) honey-blonde hair:2)...
(Mila from Kenya, long (waist-length:1.2) [black:.3] hair:2)...
(Daniela from Ecuador, long (waist-length:1.2) coffee hair:2)...
(Juana from Argentina, long (waist-length:1.2) [blue-black:.3] hair:2)...
(Harriet from England, long (waist-length:1.2) [auburn:reddish:.4] hair:2)...
(Zuzana from Czech Republic, long (waist-length:1.2) [blonde:golden-blonde:.2] hair:2)...
(Linda from Ghana, long (waist-length:1.2) [violet-black:.3] hair:2)...

This has been a great addition, and helps immensely, i just need help getting CloneCleaner to correct itself/stop repeating already performed people.
Thanks!!!!!!!!!!!!

**Comment by jackel321619** [comment]
also getting the below added to each prompt - not sure how this is happening:
(Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), (Andrea from Ecuador, long (waist-length:1.2) combed [auburn:reddish:.4] hair:2), An unparalleled.....

**Comment by MNeMoNiCuZ** [comment]
Yeah when using batches this is kind of useless unfortunately. It's always using the same seed.
Better off sticking to the normal wildcards :)
